src/main/java/org/broadinstitute/hellbender/tools/exome/ACSCopyRatioModeller.java

```diff
@@ -129,21 +135,25 @@ public ACSCopyRatioModeller(final SegmentedModel segmentedModel) {
                 final int segment = i;
                 final int targetStartInSegment = dataCollection.getStartTargetInSegment(segment);
                 final int numTargetsInSegment = dataCollection.getNumTargetsInSegment(segment);
-                final Function<Double, Double> logConditionalPDF = newMean -> {
-                    double ll = 0.;
-                    for (int target = 0; target < numTargetsInSegment; target++) {
-                        final double coverage = dataCollection.getCoveragesInSegment(segment).get(target);
-                        if (!state.getOutlierIndicator(targetStartInSegment + target)) {
-                            ll -= normalTerm(coverage, newMean, state.variance());
+                if (numTargetsInSegment == 0) {
+                    means.add(Double.NaN);
+                } else {
+                    final Function<Double, Double> logConditionalPDF = newMean -> {
+                        double ll = 0.;
+                        for (int target = 0; target < numTargetsInSegment; target++) {
+                            final double coverage = dataCollection.getCoveragesInSegment(segment).get(target);
+                            if (!state.getOutlierIndicator(targetStartInSegment + target)) {
+                                ll -= normalTerm(coverage, newMean, state.variance());
+                            }
                         }
-                    }
-                    return ll;
-                };
+                        return ll;
+                    };
 
-                final SliceSampler sampler =
-                        new SliceSampler(rng, logConditionalPDF, state.getMeanInSegment(segment),
-                                meanSliceSamplingWidth);
-                means.add(sampler.sample());
+                    final SliceSampler sampler =
+                            new SliceSampler(rng, logConditionalPDF, state.getMeanInSegment(segment),
```

Comment:
This doesn't fit on one line, but I think it would look better on two than on three.